README.md

```diff
@@ -1,2 +1,175 @@
+
+## 🛠 Built With <a name="built-with"></a>
+  -Visual Code Studio
+  <br>
+  -Github
+
+### Tech Stack <a name="tech-stack"></a>
+
+ -HTML <br>
+ -CSS
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Microverse</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Github</li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **sigma portofolio template - 2 sections**
+- **CSS styling**
+- **linters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+> 
+
+- [https://github.com/yashodhicy/Microverseday3pr](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Github account <br>
+- code editor (I use Visual code studio)
+
+
+
+
+### Setup
+
+Clone or make a copy of this repository to your desired folder: open this with your code editor.
+
+### View
+
+view this project with:
+ your browser or with a live server on visual code studio
+
+### Deployment
+
+You can deploy this project using:
+ visual code studio or any other code editor.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Yashodhi Chathurangi**
+
+- GitHub: https://github.com/yashodhicy/
+- Twitter: https://twitter.com/Yashichathucy
+- LinkedIn: https://www.linkedin.com/in/yashodhichathurangi/
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<!-- > Describe 1 - 3 features you will add to the project.
+ -->
+- [ ] **new sections will be added to html**
+- [ ] **new styles**
+- [ ] ** use more designs and tech stacks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+<!-- > Write a message to encourage readers to support your project
+ -->
+If you like this project leave a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank my code partners , my morning session teammates and code reviewers for their unconditional support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] Kindly add the `LICENSE` file to the root of your project. Presently, it does not lead to any file path which causes the broken link 👍